lib/src/assets/img/logo_nexus_generic.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="72" height="72" fill="#FFF" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You've got two copies of this again.  This is the fully optimized one, but it's in the wrong spot.  Move this to the gallery copy's location.

Reply:
> You've got two copies of this again. This is the fully optimized one, but it's in the wrong spot. Move this to the gallery copy's location.

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/a1cac62147bb4c0c2ed7ec8243455402ba1b3d1a

Reply:
> a1cac62

Oh.  I see what was going on here.  We definitely don't want the lib referencing files in the gallery.  When I wrote that I had forgotten that you had the lib hardcoded to have this icon as a fallback.  Given that you do, it did in fact need to be in the lib, not the gallery.  Now if we want to demonstrate the usage of the `logoPath` prop in the gallery, I think the gallery ought to have some other svg to refer to in the demonstration.

Reply:
> > [a1cac62](https://github.com/sonatype/sonatype-react-shared-components/commit/a1cac62147bb4c0c2ed7ec8243455402ba1b3d1a)
> 
> Oh. I see what was going on here. We definitely don't want the lib referencing files in the gallery. When I wrote that I had forgotten that you had the lib hardcoded to have this icon as a fallback. Given that you do, it did in fact need to be in the lib, not the gallery. Now if we want to demonstrate the usage of the `logoPath` prop in the gallery, I think the gallery ought to have some other svg to refer to in the demonstration.

Added a example with a custom logo https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/2a145bcec8bc245aa2789f0ab6047fa1cd56a91e